docs/scenarios_authoring/BasicNodes.md

```diff
@@ -81,11 +81,16 @@ Order is also visible on graph in edges description as a number. Be aware that l
 &nbsp;
 ## Split
 
-Split node logically splits processing into two or more parallel branches. Each branch receives all records and processes them independently.
+Split node logically splits processing into two or more parallel branches. Each branch receives all data records and processes them independently and in parallel.
+
+In the Request - Response processing mode you can use this feature to paralellize and hence speed up the processing. You must use a sequence of [Union](./BasicNodes.md#union) and [Collect](./RRDataSourcesAndSinks.md#collect) nodes to merge parallelly executed branches and collect the results from these branches. 
```

Comment:
I'd write about the semantics in RR a little bit. Maybe it'd be nice to add a note the RR has to have one sink and it implicates the usage of the Collect component to merge the parallel processing results into one element.

Reply:
added link